export async function APIERROR(msg: any) {
	return "ERROR OCCURRED WHEN FETCHING DATA";
}
